extensions/amp-mowplayer/OWNERS.yaml

```diff
@@ -1 +1,5 @@
+<<<<<<< HEAD
```

Comment:
We should remove merge conflict issues 👍

Reply:
Hi i have resolved conflicts. Could you do more review? so that we can more forward.

Reply:
Hello, thank you for fixing the merge conflicts!

I'll let @aghassemi and @alanorozco handle the rest of the review, as they are much more familiar than I am with the video players in AMP. Thank you for your contribution however! 😄